context/brand-context/default/scss/00-tokens/_icon.variables.scss

```diff
@@ -1,7 +1,7 @@
 
 // Do not edit directly
-// Generated on Fri, 21 Oct 2022 11:11:43 GMT
+// Generated on Mon, 09 Jan 2023 13:28:25 GMT
 
-$t-icon-checkbox-checked-stroke: #ffffff !default;
-$t-icon-checkbox-checked-fill: #01324b !default;
-$t-icon-select-stroke: #000000 !default;
\ No newline at end of file
+$t-icon-checkbox-checked-stroke: #ffffff;
```

Comment:
A lot of file seem to have this `!default` removed. Is this related at all to the line-height change or is this from something else and the tokens were not updated?

Reply:
it's related, when Sass meets two `!default`s for the same variable - it takes the first one, rather than the second. This meant that the overriding variable for the nature theme wasn't coming through. 

Me adding `!default` was a pre-cursor for the single release but I got it wrong here. 

Sadly I had added `!default` to pretty much every `literal` or `alias` token so we decided to remove them all, and 'circle back' to their usage later.